tests/tensor/test_slinalg.py

```diff
@@ -574,7 +578,7 @@ def test_solve_discrete_lyapunov_via_direct_complex():
     A = rng.normal(size=(N, N)) + rng.normal(size=(N, N)) * 1j
     Q = rng.normal(size=(N, N))
     X = f(A, Q)
-    assert np.allclose(A @ X @ A.conj().T - X + Q, 0.0)
+    utt.assert_allclose(A @ X @ A.conj().T - X + Q, 0.0)
```

Comment:
Any reason to use this instead of `np.testing.assert_allclose`?

Reply:
I copied from `test_kron`, which was immediately above my block of tests and used it. I'll change it.